### DISCUSSION TITLE
--install-strategy=linked

### DISCUSSION THREAD

**Discussion by pose** [npm/rfcs#658] [Question]
Hi 👋,

The new `install-strategy` argument that `npm install` introduces on npm 9 mentions a `linked` mode:

```
install-strategy

    Default: "hoisted"
    Type: "hoisted", "nested", or "shallow"

Sets the strategy for installing packages in node_modules.
hoisted (default): Install non-duplicated in top-level, and duplicated as necessary within directory structure.
nested: (formerly --legacy-bundling) install in place, no hoisting. 
shallow (formerly --global-style) only install direct deps at top-level. 
linked: (coming soon) install in node_modules/.store, link in place, unhoisted.
```

However, after doing some searching on RFCs, Pull Requests, and Issues, I haven't been able to find any references to `linked: (coming soon) install in node_modules/.store, link in place, unhoisted.`. Could you point me to where I can learn more about this new planned, upcoming feature?

Thanks,

**Comment by ruyadorno** [comment]
Could that be the name for [Isolated Mode](https://github.com/npm/rfcs/blob/e000b367d9e595bc694893c3d845df269f9b875f/accepted/0042-isolated-mode.md) ? 🤔

**Reply by JAFB321** [reply]
Yes! it is an implementation for that RFC

[Original PR](https://github.com/npm/cli/pull/6078)
